data/hai/unfettered side missions.txt

```diff
@@ -0,0 +1,531 @@
+			`	"As such, most will say that you need to prove to us you have a right to use that ship. Luckily for you, I do not believe in any of that. What matters to me is that this ship is very durable, given the time it obviously spent in the water. It also has ancient technology that would be very valuable to us."`
+			`	Their tone suddenly changes as they raise their voice for everyone to hear, "I challenge you! The winner will earn the right to own this honorable relic!"`
+			choice
+				`	"What do I get if I win?"`
+					goto win
+				`	"Can't I just sell it to you?"`
+			`	"No, because in the eyes of my brethren that would mean I, too, did not deserve this ship. They would then challenge me for it."`
+			label win
+			`	"You will win the right to access our worlds for just a meager daily fee of 500 credits. In order to keep our respect, this Ancient Ladybug may not be destroyed or sold. This is a priceless artifact. You will not be allowed to land so easily as you just did, should you refuse to prove your worth.`
+			`	"Beware, for there are no rules, other than the fact you may only use this one ship. Tell me then, what say you? Will you take to the skies, or are you too afraid?"`
+			choice
+				`	"I accept."`
+				`	"I have nothing to prove to you."`
+					decline
+			`	"Good! I will of course not be fighting you personally, as it is too risky to do so. I would also not want to deny my brethren the opportunity to prove themselves against a mighty human warrior!"`
+			`	You are not sure if they are being serious or not, but their friends seem to be in for it. Their leader approaches you. "So you think you deserve to fly this mighty ship of the Hai empire?"`
+			`	"Yeah, what makes you think that, human?" Another one adds.`
+			choice
+				`	(Say nothing.)`
+				`	"Are you trying to scare me out of the fight?"`
+					goto scare
+			`	"I guess we'll find out." Says a third.`
+			label scare
+			`	The leader gets closer and addresses you in a more serious tone, "You should be aware, <last>, that this is not a friendly competition between allies. You should see it more as a fight for you to truly earn your place with this ship, but as far as I am concerned, you are an enemy. Make no mistakes about that, this fight has only two rules: nobody other than us here may interfere, and being disabled results in losing the fight. Do not even think about tricking us by switching to something other than this Ladybug, or we will not bother showing up! You must prove your worth for this prize ship in this ship and with only this ship."`
+			`	"And do not even think about capturing our ships. Errors happen, so we can understand some may die in glorious battle, but otherwise looting us is as far as we will allow!"`
```

Comment:
```suggestion
			`	The leader gets closer and addresses you in a more serious tone. "You should be aware that this is not a friendly competition. This is a fight for you to truly earn your place with this ship. As far as I'm concerned, you are an enemy. Make no mistakes about that. This fight has only two rules: nobody other than us here may interfere, and being disabled counts as a loss. Don't even think about tricking us by switching to something other than this Ladybug, or we will not bother showing up! You must prove your worth only with the Ladybug.`
			`	"We can understand some may die in glorious battle, and to the victor go the spoils, so looting as allowed. Capturing, however, is off limits. Do so, and the whole of the Unfettered will be against you."`
```
Engineer says "there are no rules", leader says "there are two rules", but it seems like there are three: nobody can interfere, only the Ladybug, and no capturing. Which is it?

Reply:
yeah there are no official rules but he gives the implicit ones
he'll say three rules for you to be accepted among us?

Reply:
Yeah, maybe having him say something like "I know she just said no rules, but there are implicit rules. We would assume everyone knows them, but because you're an outsider I'll explain them. [list the three rules]" would be good.